src/symbolic.jl

```diff
@@ -573,80 +531,24 @@ subs(ex::Union{Expr,Symbol,Number}, d::Associative,
      funcs::Set{Symbol})
 ```
 
-Apply substituions to `ex` so that all keys in `d` are replaced by their values
+Apply substitutions to `ex` so that all keys in `d` are replaced by their values
 
 Note that the keys of `d` should be the canonical form of variables you wish to
 substitute. For example, to replace `x(1)` with `b/c` you need to have the
 entry `(:x, 1) => :(b/c)` in `d`.
-
-The one exception to this rule is that a key `:k` is treated the same as `(:k,
-0)`.
 """
 function subs(ex::Union{Expr,Symbol,Number}, d::Associative,
-              funcs::Set{Symbol})
+              funcs::Set{Symbol}=Set{Symbol}())
     _subs(ex, d, funcs)[1]
 end
 
-"""
-```julia
-subs(ex::Union{Expr,Symbol,Number}, d::Associative;
-     variables::Set{Symbol},
-     functions::Set{Symbol})
-```
-
-Verison of `subs` where `variables` and `functions` are keyword arguments with
-default values
-"""
-function subs(ex::Union{Expr,Symbol,Number}, d::Associative;
```

Comment:
At one point (if I remember correctly) you lobbied for the keyword argument versions. 

Was deleting them intentional?

Reply:
That must have been before I discovered more about optional arguments and learned that keywords args are not jitted. Does `subs(ex,d,fun=Set{}())` count as keyword argument ? ;-)
Also, we have converged to have only one option (for function names), so there is no ambiguity.